src/Components/AddTask/AddTask.tsx

```diff
@@ -1,68 +1,126 @@
 import axios from 'axios'
-import { useContext } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { useForm } from 'react-hook-form'
 import { useNavigate } from 'react-router-dom'
 import { AuthContext } from '../../Context/AuthContext'
+import { ToastContext } from '../../Context/ToastContext'
 
 export default function AddProject() {
- const navigate= useNavigate()
-const {register,handleSubmit,formState:{errors}}=useForm()
-let{baseUrl,reqHeaders}:any=useContext(AuthContext)
-const AddProject =(data)=>{
-axios.post(`${baseUrl}/Task`,data,{headers:reqHeaders}).then((response)=>{
-  
-  navigate(-1)
-  
-}).catch((error)=>{
-  console.log(error);
-  
-})
+  const navigate = useNavigate()
+  const { register, handleSubmit, formState: { errors } } = useForm()
+  let { baseUrl, reqHeaders }: any = useContext(AuthContext)
+  let { getToastValue } = useContext(ToastContext)
+  const [users, setUsers] = useState([])
+  const [allProjects, setAllProjects] = useState([])
+ 
 
-}
+
+  {/* Add  Task*/ }
+  const AddTask = (data) => {
+    axios.post(`${baseUrl}/Task`, data, { headers: reqHeaders }).then((response) => {
+   
+
+      navigate(-1)
+
+    }).catch((error) => {
+
+      getToastValue("error", error?.response?.data?.message)
+
+    })
+
+  }
+
+  {/* Get  Users*/ }
+  const Users = () => {
+    axios.get(`${baseUrl}/Users/Manager?pageSize=10&pageNumber=1`, { headers: reqHeaders }).then((response) => {
+      setUsers(response?.data?.data)
+
+    }).catch((error) => {
+      getToastValue('error', error?.response?.data?.message)
+
+    })
+  }
+
+  {/* Get  All Projects*/ }
+  const AllProjects = () => {
```

Comment:
It would be better if we call this function `getAllProjects`, because AllProjects indicates that it is a variable or some state rather than a function. This will help other developers read our code more easily.